Makefile

```diff
@@ -38,7 +38,7 @@ test:
 
 .PHONY: coverage
 coverage:
-	go test -coverprofile=cover.out -covermode=atomic -p 4 $(shell go list ./... | grep -v /tests/)
+	go test -coverprofile=cover.out -coverpkg="./..." -covermode=atomic -p 4 $(shell go list ./... | grep -v /tests/)
```

Comment:
@hunterkepley I would say lets revert this change as it doesn't seem to be doing what we expect.